econplayground/main/models.py

```diff
@@ -551,6 +521,38 @@ def clone(self):
         return c
 
 
+class Question(models.Model):
+    title = models.TextField(max_length=1024, default='Untitled')
+    tags = models.ManyToManyField(Assignment, blank=True)
```

Comment:
I don't know if adding an Assignment as a tag really makes sense, conceptually. In the past we've implemented tagging on models using the [django-taggit](https://django-taggit.readthedocs.io/en/latest/index.html) library, e.g.: https://github.com/ccnmtl/dmt/blob/master/dmt/main/models.py#L1254

If our intention here is to tie the questions to the Assignment in a many-to-many relation, then let's just call this field `assignments`. I think that would be redundant though, as we can already infer all assignments tied to a given Question via the QuestionBank's ForeignKey to Assignment.

Reply:
Actually I think I can remove the tag or assignments property entirely from the Questions model. I looked deeper into the ManyToMany relationship when I was working out a symmetrical linking issue between question banks and I found out that ManyToMany relationships are symmetrical by default. Meaning that the Question models can already do a search of the Question Banks which they are connected to, thus they can access the Assignment data as well without adding anything further to the model. 

I'll make the changes and push an update to the PR